### DISCUSSION TITLE
Add EmojiDjango

### DISCUSSION THREAD

**Discussion by alphacodes26** [ScratchAddons/ScratchAddons#3920] [Ideas]
Do I really have to be a third-party to do this? If I can, Introducing, EmojiDjango! They are on a mission to add more emojis to the forums. Here's the link: [https://scratch.mit.edu/discuss/topic/557083/](https://scratch.mit.edu/discuss/topic/557083/)

**Comment by CST1229** [comment]
*Emodjiango
Also, this could work as an addon to the emoji picker addon or to the forum toolbar addon.

**Reply by WorldLanguages** [reply]
django but the j is silent

**Reply by CST1229** [reply]
> django but the j is silent

🍡

**Reply by retronbv** [reply]
:shipit:

**Comment by kccuber-scratch** [comment]
Theres a GitHub repository for it too, [lopste/emodjiango](/lopste/emodjiango)
